build.gradle

```diff
@@ -72,6 +72,13 @@ tasks.withType(Test) {
 }
 
 test {
+  jvmArgs = ["-agentlib:jdwp=transport=dt_socket,server=y,address=5005,suspend=y"]
```

Comment:
Does this need to be merged or is it only for local uses?

Reply:
Good point. I'll remove it for now